gatsby/plonerestapi.rst

```diff
@@ -0,0 +1,131 @@
+Fetching Data Using Plone REST.API
+==================================
+
+All the data from a Plone site is available in the JSON format using the `plone.restapi <https://plonerestapi.readthedocs.io/en/latest/introduction.html>`_.
+
+We'll be working a lot with this API while working on the Gatsby source-plugin, so it is recommended that you have an API browser to explore the API.
+Install `Postman <https://www.getpostman.com/>`_ and do go through the quick guide to working with plone.restapi:
+
+https://plonerestapi.readthedocs.io/en/latest/exploring.html#exploring-api-postman-onboarding
+
+.. note:: We're using the same endpoints as for loading the site on a browser but setting the header `Accept: application/json` is what retrieves JSON data for us.
+
+Exploring The Plone REST.API
+----------------------------
+
+We'll be using https://plonedemo.kitconcept.com/en as our source Plone site, since it's already been configured with the plone.restapi and is all ready for our usage.
+
+Let's start with the root itself: A GET Request to `https://plonedemo.kitconcept.com/en` would give the JSON data for the root of the Plone site.
```

Comment:
Shorter sentences, grammar, and syntax fixes:

```
Let us start with the root itself.
Send a ``GET`` request to https://plonedemo.kitconcept.com/en.
This will return the JSON data for the root of the Plone site.
```